src/client/components/app/Transactions/EnableSpigotTx.tsx

```diff
@@ -88,11 +93,14 @@ export const EnableSpigotTx: FC<EnableSpigotTxProps> = (props) => {
   const [settingOwnerSplit, setOwnerSplit] = useState('100');
   const [settingClaimFunc, setClaimFunc] = useState('');
   const [settingTransferFunc, setTransferFunc] = useState('');
+  const [revenueContractAdd, setRevenueContractAdd] = useState<string>('');
+  const [revContractABI, setRevenueContractABI] = useState(false);
+  const [funcType, setFuncType] = useState({ id: '1', label: 'Repay from:', value: 'Wallet' });
```

Comment:
The values for the keys (label & value) should be reworded to fit new use of dropdown.